packages/apps/docs/src/components/Layout/Full/index.stories.tsx

```diff
@@ -30,12 +30,8 @@ const aSideMenuTree = [
         tag: 'h3' as unknown as TagNameType,
         title: 'Kadena raises $2.25M in first funding round',
         children: [],
-        index: 1,
```

Comment:
was needed for hashes of headers.
But this was refactored 2 months ago. not needed anymore